package.json

```diff
@@ -1,17 +1,18 @@
 {
   "name": "babel-plugin-lightscript",
-  "version": "0.3.2",
+  "version": "0.4.0",
   "description": "Compile LightScript to JavaScript.",
   "main": "index.js",
   "scripts": {
     "test": "node node_modules/.bin/mocha test",
+    "test:debug": "node --inspect --debug-brk node_modules/.bin/_mocha test",
```

Comment:
No, they would need to be inlined at build time for them to work, we could do a separate debug build?